kolibri/core/templates/kolibri/base.html

```diff
@@ -1,6 +1,7 @@
 {% load i18n kolibri_tags webpack_tags js_reverse cache %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
```

Comment:
Not sure about the properties of this tag, so let's leave it here for now - but we may want to add it dynamically from within the audio component.